build/source/dshare/get_ixname.f90

```diff
@@ -21,6 +21,7 @@
 module get_ixname_module
 ! used to get the index of a named variable
 USE nrtype                                          ! variable types, etc.
+USE multiconst,only:integerMissing
```

Comment:
I consolidated these throughout, however note that it was necessary to put them all in `multiconst.f90` because if they went in `globalData.f90` then there would be recursive dependency with `var_lookup.f90`.